ch.theologeek.Manuskript.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Tobias Frisch -->
```

Comment:
file is missing oars rating

Reply:
At line 7 it is but I have added it with the second commit. Maybe that's why it was missing.